tests/Owin.Security.OpenIdConnect.Server.Tests/OpenidConnectTestServer.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Owin;
+using Microsoft.Owin.Testing;
+using Newtonsoft.Json.Linq;
+
+namespace Owin.Security.OpenIdConnect.Server.Tests
+{
+    public class OpenIdConnectTestServer : TestServer
```

Comment:
I wonder if we couldn't avoid this class and directly use `TestServer` with the minimal OIDC options needed to pass the different tests.